src/libraries/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -5210,7 +5274,7 @@ public new abstract partial class X64 : System.Runtime.Intrinsics.X86.Sse3.X64
             public static new bool IsSupported { get { throw null; } }
         }
     }
-    
+
```

Comment:
Yes, but also not entirely...

Several of the corelib facades have bits that weren't done by the tool (often attributes or simple ordering things) and which make generating this "properly" result in large diffs.

So, I typically cut out just the new bits from the generated file and paste them into the checked in copy to avoid that issue.

We should probably do the work to ensure they are in sync again, but it hasn't bubbled up in priority yet.

Reply:
https://github.com/dotnet/runtime/issues/83166